### PR TITLE
Aleksei/fix multisendmessage

### PR DESCRIPTION
**Pull request by iambeone**
Closes #ISSUE

**Description:**

Fixed multiSendMessage

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

File: src/components/transactions/message-view/MultiSendMessageDetails.vue [hunk 1]
```diff
@@ -60,12 +55,35 @@ export default {
     }
   },
   computed: {
+    recipient() {
+      let index = this.transaction.value.inputs.findIndex(
+        input => input.address == this.sessionAddress
```
Comment (Bitcoinera) [inline src/components/transactions/message-view/MultiSendMessageDetails.vue]: I didn't know this `findIndex` function. Wow, looks powerful!

File: src/components/transactions/message-view/MultiSendMessageDetails.vue [hunk 2]
```diff
@@ -60,12 +55,35 @@ export default {
     }
   },
   computed: {
+    recipient() {
```
Comment (faboweb) [inline src/components/transactions/message-view/MultiSendMessageDetails.vue]: what are you doing here? why the conditional that the user address is in the inputs?

File: src/components/transactions/message-view/MultiSendMessageDetails.vue [hunk 3]
```diff
@@ -60,12 +55,35 @@ export default {
...
+      let index = this.transaction.value.inputs.findIndex(
+        input => input.address == this.sessionAddress
+      )
+      if (index !== undefined && this.transaction.value.outputs[index]) {
+        return this.transaction.value.outputs[index].address
+      }
+      return false
+    },
+    sender() {
```
Comment (faboweb) [inline src/components/transactions/message-view/MultiSendMessageDetails.vue]: same here, why is the sender necessarily in the outputs?

Comment (iambeone) [reply src/components/transactions/message-view/MultiSendMessageDetails.vue]: Only my thoughts... Like if the user is in inputs (e.g. senders) he is a sender, if he is in outputs (e.g. recipients) he is a recipient

Comment (Bitcoinera) [reply src/components/transactions/message-view/MultiSendMessageDetails.vue]: Isn't it actually the opposite?

Comment (iambeone) [reply src/components/transactions/message-view/MultiSendMessageDetails.vue]: True, my fault, but the code is right) We are looking for a sender if our address is a recipient and we are looking for a recipient if our address is sender.

Comment (faboweb) [reply src/components/transactions/message-view/MultiSendMessageDetails.vue]: I get it. Deserves comments though

**Comment by faboweb** [comment]
A lot of effort for us wanting to abstract transactions

**Comment by faboweb** [comment]
Should we just drop this? Mario is already working on the abstraction

**Comment by Bitcoinera** [comment]
> Should we just drop this? Mario is already working on the abstraction

What do you think @iambeone ? Seems reasonable

**Pull request closed without merging**